src/Build/Resources/xlf/Strings.cs.xlf

```diff
@@ -149,8 +149,7 @@
       </trans-unit>
       <trans-unit id="LoadingProjectCachePlugin">
         <source>Loading the following project cache plugin: {0}</source>
-        <target state="needs-review-translation">Načítá se následující modul plug-in mezipaměti projektu:
```

Comment:
@cristianosuzuki77 we're not at all sure what happened here. Is there any reason why we'd get a newline here when we didn't change anything?